src/nav/AppWithNavigation.js

```diff
@@ -2,10 +2,17 @@
 
 import { createReduxContainer } from 'react-navigation-redux-helpers';
 
-import { connect } from '../react-redux';
+import { connect } from 'react-redux';
 import { getNav } from '../selectors';
 import AppNavigator from './AppNavigator';
 
-export default connect(state => ({
-  state: getNav(state),
-}))(createReduxContainer(AppNavigator, 'root'));
+// $FlowFixMe - should use a type-checked `connect`
+export default connect(
+  state => ({
+    state: getNav(state),
+  }),
+  null,
+  null,
+  { forwardRef: true },
+  // $FlowFixMe - not sure what's going on here
+)(createReduxContainer(AppNavigator, 'root'));
```

Comment:
I think this takes care of it:

```suggestion
)((createReduxContainer(AppNavigator, 'root'): ComponentType<{||}>));
```

Here's what the error message was:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/nav/AppWithNavigation.js:16:3

Missing type annotation for ContainerProps. ContainerProps is a type parameter
declared in function [1] and was implicitly instantiated at call of
createReduxContainer [2].

     src/nav/AppWithNavigation.js
     13│   null,
     14│   null,
     15│   { forwardRef: true },
 [2] 16│ )(createReduxContainer(AppNavigator, 'root'));
     17│

     node_modules/react-navigation-redux-helpers/src/create-redux-container.js
 [1] 28│ function createReduxContainer<
     29│   State: NavigationState,
     30│   Options: {},
     31│   NavigatorProps: NavigationNavigatorProps<Options, State>,
     32│   NavigatorType: NavigationNavigator<
     33│     State,
     34│     Options,
     35│     NavigatorProps,
     36│   >,
     37│   ContainerProps: {
     38│     ...$Diff<NavigatorProps, InjectedProps<State>>,
     39│     ...$Exact<RequiredProps<State>>,
     40│     theme: SupportedThemes | 'no-preference',
     41│   },
     42│ >(
     43│   Navigator: NavigatorType,
     44│   key?: string = "root",
     45│ ): React.ComponentType<ContainerProps> {
```
So it can't tell what the type parameter `ContainerProps` should be for this `createReduxContainer` call. And that's apparently the props type of the return value. Adding an annotation with that return type fixes the issue.

I suspect the reason this appeared with this change is that before this change, with a well-typed `connect`, the type on `connect` propagated information through so that Flow could infer the type of its argument here, the return value of `createReduxContainer`. And then when `connect` is marked with a fixme, Flow just doesn't know what to think here.